SPEC.md

```diff
@@ -857,7 +869,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
 }
 ```
 
-* **acVersion** (string, required) represents the version of the schema spec (must be in [semver](http://semver.org/) format)
+* **acVersion** (string, required) represents the version of the [schema specification](#ac-version-type) (must be in [semver](http://semver.org/) format)
```

Comment:
nits:
i) please change the link part of the text to `[version of the schema specification]`
ii) could you please swap this line with the next? they are out of order with the imagemanifest :( (not your fault)